app/src/androidTest/java/de/test/antennapod/feed/FeedItemTest.java

```diff
@@ -0,0 +1,35 @@
+package de.test.antennapod.feed;
+
+import android.test.AndroidTestCase;
+import de.danoeh.antennapod.core.feed.FeedItem;
+
+public class FeedItemTest extends AndroidTestCase {
+
+    public void testShownoteLength() throws Exception {
```

Comment:
Can you either split these into separate test cases (name should reflect the behaviour) or parameterise the test?
There is a lot of duplication here (`"Hello world"`, assertions are all the same etc.)